### ISSUE TITLE
Accessability: Verify Keyboard support

### ISSUE THREAD

**Issue by crutkas** [Microsoft/WindowsTemplateStudio#495]
keyboard support works for everything. Adding, removing, and navigating both wizard and templates.

**Comment by mvegaca** [comment]
New branches:
- [version1.3](https://github.com/Microsoft/WindowsTemplateStudio/tree/version1.3)
-  [accessability](https://github.com/Microsoft/WindowsTemplateStudio/tree/accessiblity)
- [Issue495-keyboard](https://github.com/Microsoft/WindowsTemplateStudio/tree/Issue495-keyboard)

All accessability Issues will be merged to accessability branch.
Stable accessibility commits will be merged to version1.3 branch.

**Comment by mvegaca** [comment]
I've removed Local Keyboard Navigation on ListView that allow user to select on single element. Following [commit](https://github.com/Microsoft/WindowsTemplateStudio/commit/e77423d391726a03490f58967271fe11dd3a3cd9)
- Project Type
- Project Framework
- New feature / page

**Comment by mvegaca** [comment]
What is the best approach to implement drag and drop using the keyboard?
I'm trying to move files on VisualStudio simulating drag and drop and I can't do it, I can only do it using context menu options.

It makes sense think about using key shortcuts to reorder pages using the keyboard?
(i.e. space + arrow up, space + arrow down)

**Comment by mrlacey** [comment]
> New branches:
> 
> version1.3
> accessability
> Issue495-keyboard

Why these three new branches?
Are they all for this issue?
What is the purpose of each?
Are you proposing changing the workflow by creating "version1.3"?
What, if anything, should anyone else need to know about them?
What's wrong with dev branch?

**Comment by crutkas** [comment]
accessability right now was me going though and commenting stuff a while back and didn't want to lose anything.  1.3 is the branch while we ship 1.2 so Madrid can continue working.  At this point we just merged into master so we _could_ do a hotfix branch off that and merge to master then master back into dev.

#495 branch, no clue

**Comment by crutkas** [comment]
1.3 is a temp branch that will be merged back into dev soon

**Comment by mvegaca** [comment]
My idea was to merge al accessability changes to accessability branch. Accessability borns from version1.3 that will be merged to dev. David and me will be working on different accessability Issues.

**Comment by crutkas** [comment]
1.3 should be our major accessibility push. IMO, kill the accessibility branch as it only has a few notes I think from my playing with colors.

For any change, please lets chat to be sure the solution is correct as we'll be doing some global impact stuff and i think we may want to do some ordering of work like making some items controls so we don't redo work over and over again.

**Comment by mvegaca** [comment]
Updated on #812

**Comment by mvegaca** [comment]
I was testing full templates configured app on different project types and frameworks, all pages and features are fully accessible using the keyboard also when the theme is switched.

**Comment by mvegaca** [comment]
This issue was pending on #537. Se can close out soon.
